docs/modules/ROOT/pages/documentation/payara-server/upgrade-server.adoc

```diff
@@ -0,0 +1,67 @@
+[[Upgrade Server]]
+= Upgrade Payara Server
+
+Payara Server can be upgraded to new versions from version *5.24.0*
```

Comment:
There probably needs to be a warning or note stating that the feature is a technical preview.